controllers/tokenupload_controller.go

```diff
@@ -132,17 +132,19 @@ func (r *TokenUploadReconciler) reconcileRemoteSecret(ctx context.Context, uploa
 	return nil
 }
 
+var uploadSecretSelector = metav1.LabelSelector{
```

Comment:
Style-nazi: we usually define global vars at the top of the file.

Reply:
Good point. I will move it to the top with SetupWithManager as I wanted to keep to close to it and we usually put the setup at the top anyway.